### ISSUE TITLE
Incorrect TextView wrap

### ISSUE THREAD

**Issue by novalu** [google/flexbox-layout#62]
I'm using FlexboxLayout 0.2.0, but I think that text is still not wrapping correctly.

```
<com.google.android.flexbox.FlexboxLayout
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:padding="16dp"
    app:flexWrap="wrap">

    <TextView
        android:id="@+id/text1"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:textSize="16sp"
        android:text="AAAAAAB"/>

    <TextView
        android:id="@+id/text2"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:textSize="16sp"
        android:text="CCCCCCCCCCCCCCCCCCCCCCCC"/>

</com.google.android.flexbox.FlexboxLayout>
```

![flex](https://cloud.githubusercontent.com/assets/6546116/15760973/133386e8-2916-11e6-949f-aaf4729f4b41.png)

This layout is rendered as top image, then I added 'D' (middle image), then another 'D' (bottom image). In top and bottom render `FlexboxLayout` wrapped text correctly, but in middle render there are missing 'B' from `text1` and 'D' from `text2`. 

When changed attribute `layout_width` to `wrap_content`, behavior is different, but still incorrect.

![flexb](https://cloud.githubusercontent.com/assets/6546116/15761348/f8e7b50a-2917-11e6-96b7-54d17c12fdde.png)

In first image is the same text as above, then added 'E', another 'E' and once again 'E'. In first and last is text rendered correctly. In second image there is missing half size of char 'E', in third image are missing half size of first char 'E' and completely missing second char 'E'.

After I've changed `android:padding` to `android:layout_margin`, text is rendering correctly in all of the above cases.

**Comment by thagikura** [comment]
Hi, thank you for your report.

I spot the code that affects this behavior, but my intuition says fixing it is not as easy as it looks.
Need to consider the measurements for views with LayoutParams.WRAP_CONTENT differently from LinearLayout since the FlexboxLayout has the concept of wrapping.

**Comment by thagikura** [comment]
This should be fixed by #70 

I think this is a edge case hard to find. Thank you reporting this!

**Comment by thagikura** [comment]
Actually #70 wasn't a right fix. Reverted it and submitted #74 instead.
Anyway I'm going to include this fix in the next version.
